src/query.rs

```diff
@@ -52,4 +52,46 @@ impl Query {
             inner: Box::new(inner),
         })
     }
+
+    /// Construct a Tantivy's FuzzyTermQuery
+    #[staticmethod]
+    #[pyo3(signature = (schema, field_name, text, distance = 1, transposition_cost_one = true))]
+    pub(crate) fn fuzzy_term_query(
+        schema: &Schema,
+        field_name: &str,
+        text: &PyString,
+        distance: u8,
+        transposition_cost_one: bool,
+    ) -> PyResult<Query> {
+        let term = make_term(&schema.inner, field_name, &text)?;
+        let inner = tv::query::FuzzyTermQuery::new(
+            term,
+            distance,
+            transposition_cost_one,
+        );
+        Ok(Query {
+            inner: Box::new(inner),
+        })
+    }
+
+    /// Construct a Tantivy's FuzzyTermQuery of the term prefix
+    #[staticmethod]
+    #[pyo3(signature = (schema, field_name, text, distance = 1, transposition_cost_one = true))]
+    pub(crate) fn fuzzy_term_query_prefix(
```

Comment:
I think a Python API would rather use a `prefix = False` keyword argument instead of a separate method here.